ocaml/typing/typeclass.ml

```diff
@@ -417,7 +417,7 @@ and class_type_aux env virt self_scope scty =
                                                    List.length styl)));
       let ctys = List.map2
         (fun sty ty ->
-          let cty' = transl_simple_type env ~closed:false Alloc.Const.legacy sty in
+          let cty' = transl_simple_type ~new_var_jkind:Sort env ~closed:false Alloc.Const.legacy sty in
```

Comment:
I think (quite low confidence) that this should be `Any`. A class type can appear as an annotation on a class expression (where `Any` is appropriate) or on the RHS of a `class type` definition. In the latter case, the class type is very like a manifest: any free type variables will be in scope elsewhere.